docs/tutorials/proposals/index.md

```diff
@@ -15,12 +15,21 @@ import Topic from '/docs/topics/_topic-governance.mdx'
 
 Governance with tokenholder participation is essential on Vega. Markets and assets can only be added through governance, and network parameters, markets and assets all must be amended through governance as well.
 
+## Requirements
+
+You must meet certain criteria before you can submit. Requirements may differ depending on the type of proposal.
+
+You will typically need:
+
+- A connected [Vega wallet](/docs/tools/vega-wallet/index.md), with your wallet name and public key to hand
+- A minimum amount of Vega tokens associated with that public key
```

Comment:
```suggestion
- A minimum amount of Vega tokens associated with that public key. The amount depends on the proposal type
```

Reply:
The sentence just above this says "Requirements may differ depending on the type of proposal". Should we say it there, or here?

Reply:
Oh yes, forgot about that. Just at the top is fine - if it turns out to be unclear it can move.